# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/moz.build
# Commit: dabd7d6836c8
# Full Hash: dabd7d6836c8ed368c06537c3c19a61295d67a18
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2024-06-21 04:13:05
# Regressor Bug: 1728331
# File Overlap Count: 1
# Description:
#   Bug 1728331 - Part 4: Make ContentParent KeepAlives explicit with RAII references, r=smaug,dom-worker-reviewers,asuth
#   
#   This is a fairly significant patch, however it would be difficult to break it
#   down into smaller patches:
#   
# ==============================================================================

diff -r 9e04f49c926e -r dabd7d6836c8 dom/ipc/moz.build
--- a/dom/ipc/moz.build	Thu Jun 20 19:24:51 2024 +0000
+++ b/dom/ipc/moz.build	Thu Jun 20 19:24:51 2024 +0000
@@ -85,6 +85,7 @@
     "TabContext.h",
     "TabMessageTypes.h",
     "TabMessageUtils.h",
+    "UniqueContentParentKeepAlive.h",
     "URLClassifierChild.h",
     "URLClassifierParent.h",
     "UserActivationIPCUtils.h",
@@ -148,6 +149,7 @@
     "SharedStringMap.cpp",
     "StructuredCloneData.cpp",
     "TabContext.cpp",
+    "UniqueContentParentKeepAlive.cpp",
     "URLClassifierParent.cpp",
     "WindowGlobalActor.cpp",
     "WindowGlobalChild.cpp",